tests/unit/ng/events/test_sre_event.py

```diff
@@ -0,0 +1,39 @@
+# pylint: disable=missing-docstring
+# pylint: disable=protected-access
+# pylint: disable=too-few-public-methods
+# pylint: disable=redefined-slots-in-subclass
+
+
+from logprep.ng.abc.event import Event
+from logprep.ng.event_state import EventState, EventStateType
+from logprep.ng.events.sre_event import SreEvent
+from tests.unit.ng.test_event import TestEventClass
+
+
+class DummyEvent(Event):
+    __slots__ = Event.__slots__
+
+
+class TestSreEvents(TestEventClass):
+
+    def test_sre_event_initializes_correctly(self) -> None:
```

Comment:
what does "correctly" means? ;)

```suggestion
    def test_sre_event_initialisation(self) -> None:
```